### DISCUSSION TITLE
Contex length for Mistral

### DISCUSSION THREAD

**Discussion by rengecky** [oobabooga/text-generation-webui#5393] [Q&A]
According to the default configs of the Mistral model, max_seq_len is set to 32k, although the model has only 8k context.  Can someone explain to me how max_seq_len differs from context?

**Comment by TechScorpion** [comment]
I assume it can output up to 32K tokens (max_seq_len) - ie. the text it generates for the user, but only have as input (context. maybe that might also include the prompt) up to 8K tokens.
